src/javascripts/cookies.js

```diff
@@ -0,0 +1,43 @@
+// Add cookie to session
+
+(function() {
+  var
+    cookie = false,
+    cookieContent = document.getElementById('cookie');
+
+  checkCookie();
+  if (cookie === true) {
+    cookieContent.style.display = 'none';
+  } else {
+    cookieContent.style.display = 'block';
+  }
+
+  function setCookie(cname, cvalue, exdays) {
+    var d = new Date();
+    d.setTime(d.getTime() + (exdays * 24 * 60 * 60 * 1000));
+    var expires = 'expires=' + d.toUTCString();
+    document.cookie = cname + '=' + cvalue + '; ' + expires;
+  }
+
+  function getCookie(cname) {
+    var name = cname + '=';
+    var ca = document.cookie.split(';');
+    for (var i = 0; i < ca.length; i++) {
+      var c = ca[i].trim();
+      if (c.indexOf(name) === 0) return c.substring(name.length, c.length);
+    }
+    return '';
```

Comment:
Might be more useful to return false rather than a empty string - though falsy, triple operators will be more succinct in what's actual gone wrong here for debugging.